agreement.md

```diff
@@ -0,0 +1,151 @@
+# Aragon Network Agreement
+
+## Section 0. Background
+The Aragon Network is a Decentralized Autonomous Organization that builds and maintains a dispute resolution protocol and other smart contract infrastructure.
+
+The Aragon Network is governed by Aragon Network Token (ANT) holders, who supported the initial creation of the network and under this Agreement will continue to guide its evolution through established governance processes.
+
+This Agreement exists to enumerate the shared mission, principles, and governance guidelines that bind ANT holders in their collaborative effort to create value for the Aragon Network.
```

Comment:
Simply "This Agreement describes the..." or "...specifies the..." or "...establishes the..."?